js/src/wp-seo-replacevar-plugin.js

```diff
@@ -127,6 +130,60 @@ import "./helpers/babel-polyfill";
 		}
 	};
 
+	/**
+	 * Subscribes to Gutenberg to watch a possible parent page change.
+	 *
+	 * @return {void}
+	 */
+	YoastReplaceVarPlugin.prototype.subscribeToGutenberg = function() {
+		if ( ! isGutenbergDataAvailable() ) {
+			return;
+		}
+
+		let fetchedParents = { 0: "" };
+		let currentParent  = null;
+		const wpData       = wp.data;
+		wpData.subscribe(
+			function() {
```

Comment:
Use a anonymous function ( `() => {}` ). This also removes the need for the `.bind()` at the end.